.github/workflows/ci.yml

```diff
@@ -115,3 +119,34 @@ jobs:
       
       - name: Run build
         run: npm run build
+
+  build-docker-image:
+    runs-on: ubuntu-latest
+    needs: [test, fe-lint-build]
+    steps:
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
```

Comment:
update to v2?
see https://github.com/aws-actions/amazon-ecr-login?tab=readme-ov-file#new-v2-release